README.md

```diff
@@ -1,4 +1,4 @@
-# Welcome to Authentication daemon for external Broker
+# Canonical authd
```

Comment:
Not sure about this change.
The project's organisation is Ubuntu. Then the title doesn't bring any information just the name of the project and the owner. 

Maybe "authd: an **auth**entication **d**aemon for cloud identity providers"

It is then redundant with the first sentence of the first paragraph which could be removed if we don't want to repeat the statement but it doesn't hurt to keep it.

Reply:
I like your suggestion.
The product name should be in the header but the org is not needed.